js/services/Poll.js

```diff
@@ -55,5 +61,5 @@ define(['knockout', 'visibilityjs'], (ko, Visibility) => {
     }
   }
 
-  return new PollService();
+  return PollService;
```

Comment:
This change is required since we want to extend base class, am I right?
But should this be done without breaking current functionality?